### DISCUSSION TITLE
What is the status of uuid v7?

### DISCUSSION THREAD

**Discussion by mbrimmer83** [uuidjs/uuid#746] [Q&A] [answered]
I noticed this [pr](https://github.com/uuidjs/uuid/pull/681) has been sitting for a little while. Is uuid v7 still experimental and this is why the library doesn't yet support it?

[Edit: FWIW, `uuid@10` has been released with full support for RFC9562.]

**Comment by broofa** [comment, marked as answer]
That, and @ctavan and I have had limited time to devote to maintaining this project.

**Reply by mbrimmer83** [reply]
@broofa Thanks for the update and all the work you do on the library. I'm starting to see more chatter about various teams using v7. We use this library heavily and would like to use v7 for a project we have going out this fall. 🤞

**Reply by todor0v** [reply]
@broofa Just FYI that the new version has been approved now. ([Link](https://www.rfc-editor.org/rfc/rfc9562))

**Comment by masb0ymas** [comment]
i can't use uuid@10, because @types/uuid is not supported for this version.
should I report this issue to the [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/uuid) Repository project?

**Reply by broofa** [reply]
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/69773
